src/lib/remark-plugins/remarkEmbed.ts

```diff
@@ -33,6 +33,44 @@ export const oEmbedTransformer: Readonly<Transformer> = {
   },
 }
 
+export const googleSlidesTransformer: Readonly<Transformer> = {
+  hName: "iframe",
+  hProperties: async (url): Promise<HProperties> => {
+    const isWeb = url.pathname.startsWith("/presentation/d/e/")
+    if (isWeb) {
+      // Webとして公開されている場合は、そのまま埋め込み用のURLを返す
```

Comment:
Google スライドの機能に詳しくない人の目線でコメントが若干分かりにくいと感じたので、以下のように変更すると良いと思います。

```suggestion
      // [ファイル] > [共有] > [ウェブに公開]で生成されたリンクは、そのまま埋め込み用のURLを返す
```

https://support.google.com/docs/answer/183965